spec/controllers/posts_controller_spec.rb

```diff
@@ -944,7 +975,7 @@
       end
 
       it "works for unread" do
-        third_reply = post.replies.ordered.limit(3).last
+        third_reply = post.replies.ordered.limit(4).last
```

Comment:
*squints* I think this is right? Looks a bit confusing, though, maybe we should just do:
```rb
replies = post.replies.ordered.to_a
third_reply = replies[3]
second_last_reply = replies[-2]
```
or something like that?